src/data/machines.js

```diff
@@ -46,22 +46,126 @@ export const gpuTypes = [
+  { name: 'ASIA-NORTHEAST3-B', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-NORTHEAST3-C', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-SOUTH1-A', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-SOUTH1-B', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-SOUTH1-C', validTypes: [] },
+  { name: 'ASIA-SOUTHEAST1-A', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-SOUTHEAST1-B', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-p4'] },
+  { name: 'ASIA-SOUTHEAST1-C', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-p4'] },
+  { name: 'ASIA-SOUTHEAST2-A', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-SOUTHEAST2-B', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-SOUTHEAST2-C', validTypes: [] },
+  { name: 'AUSTRALIA-SOUTHEAST1-A', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-p4'] },
+  { name: 'AUSTRALIA-SOUTHEAST1-B', validTypes: ['nvidia-tesla-p4'] },
+  { name: 'AUSTRALIA-SOUTHEAST1-C', validTypes: ['nvidia-tesla-p4'] },
+  { name: 'EUROPE-NORTH1-A', validTypes: [] },
+  { name: 'EUROPE-NORTH1-B', validTypes: [] },
+  { name: 'EUROPE-NORTH1-C', validTypes: [] },
+  { name: 'EUROPE-WEST1-B', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-p100', 'nvidia-tesla-k80'] },
+  { name: 'EUROPE-WEST1-C', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'EUROPE-WEST1-D', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-p100', 'nvidia-tesla-k80'] },
+  { name: 'EUROPE-WEST2-A', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'EUROPE-WEST2-B', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'EUROPE-WEST2-C', validTypes: [] },
+  { name: 'EUROPE-WEST3-A', validTypes: [] },
+  { name: 'EUROPE-WEST3-B', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'EUROPE-WEST3-C', validTypes: [] },
+  { name: 'EUROPE-WEST4-A', validTypes: ['nvidia-tesla-v100', 'nvidia-tesla-p100'] },
+  { name: 'EUROPE-WEST4-B', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100', 'nvidia-tesla-p4'] },
+  { name: 'EUROPE-WEST4-C', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100', 'nvidia-tesla-p4'] },
+  { name: 'EUROPE-WEST6-A', validTypes: [] },
+  { name: 'EUROPE-WEST6-B', validTypes: [] },
+  { name: 'EUROPE-WEST6-C', validTypes: [] },
+  { name: 'NORTHAMERICA-NORTHEAST1-A', validTypes: ['nvidia-tesla-p4'] },
+  { name: 'NORTHAMERICA-NORTHEAST1-B', validTypes: ['nvidia-tesla-p4'] },
+  { name: 'NORTHAMERICA-NORTHEAST1-C', validTypes: ['nvidia-tesla-p4'] },
+  { name: 'SOUTHAMERICA-EAST1-A', validTypes: [] },
+  { name: 'SOUTHAMERICA-EAST1-B', validTypes: [] },
+  { name: 'SOUTHAMERICA-EAST1-C', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'US-CENTRAL1-A', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100', 'nvidia-tesla-p4', 'nvidia-tesla-k80'] },
+  { name: 'US-CENTRAL1-B', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100'] },
+  { name: 'US-CENTRAL1-C', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100', 'nvidia-tesla-p100', 'nvidia-tesla-p4', 'nvidia-tesla-k80'] },
+  { name: 'US-CENTRAL1-F', validTypes: ['nvidia-tesla-v100', 'nvidia-tesla-p100'] },
```

Comment:
I do see a `T4` for this one:

```suggestion
  { name: 'US-CENTRAL1-F', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100', 'nvidia-tesla-p100'] },
```

Reply:
Fixed